zbus_macros/src/iface.rs

```diff
@@ -852,6 +865,9 @@ pub fn expand(args: Punctuated<Meta, Token![,]>, mut input: ItemImpl) -> syn::Re
             async fn get(
                 &self,
                 property_name: &str,
+                s: &#zbus::ObjectServer,
+                c: &#zbus::Connection,
+                h: &Option<#zbus::message::Header<'_>>,
```

Comment:
I think it was a bad idea to choose such short non-descriptive names in the beginning but now that we're getting more args, it's even worse now. You're just following the existing code, so not at all your fault but if could you use the opportunity to change the code to use more meaningful names, that would be awesome. Bonus points for first updating the existing use in a separate commit that comes before this one. :wink: 

```suggestion
                object_server: &#zbus::ObjectServer,
                c: &#zbus::Connection,
                h: &Option<#zbus::message::Header<'_>>,
```